### PR TITLE
Refactor GraphQL query in client side for deleteAddressSpace and deleteAddress to send many target objects 

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Refactoring

### Description

Refactor GraphQL client query for deleteAddressSpace and deleteAddress to pass variable as an array argument in console UI

### Checklist

- [ ] Update/write design documentation in ./documentation/design
 - [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ]  Check RBAC rights for Kubernetes / OpenShift roles
- [ ]  Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ]  Reference relevant issue(s) and close them after merging
- [ ]  Update CHANGELOG.md

### CODE REVIEW

**Comment by anukritijha** [comment]
@drcoolsanjeev Please update the template.

File: console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx [hunk 1]
```diff
@@ -100,12 +68,19 @@ export default function AddressSpacePage() {
 
   const onConfirmDeleteAll = async () => {
     if (selectedAddressSpaces && selectedAddressSpaces.length > 0) {
-      const data = selectedAddressSpaces;
-      await Promise.all(
-        data.map((addressSpace, index) =>
-          deleteAddressSpace(addressSpace, index)
-        )
+      let queryVariables: Array<{ name: string; namespace: string }> = [];
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx:71]: I think, deleteAddressSpaceVariables makes more sense, wdyt?

Comment (anukritijha) [reply console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx:71]: And similar changes in the other query variables as well.

Comment (drcoolsanjeev) [reply console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx:71]: This is just a function scope variable and not a global scope variable. So i think queryVaraible is fine.

**Comment by k-wall** [comment]
@drcoolsanjeev  As #4567 is now merged, I took the liberty of rebasing this PR and have kicked off the tests.

**Comment by k-wall** [comment]
@enmasse-ci run tests testcase=FirefoxConsoleTest

File: console/console-init/ui/src/graphql-module/queries/AddressSpace.ts [hunk 2]
```diff
@@ -9,8 +9,8 @@ import { removeForbiddenChars } from "utils";
 import { generateFilterPattern } from "./query";
 
 const DELETE_ADDRESS_SPACE = gql`
-  mutation delete_as($a: ObjectMeta_v1_Input!) {
-    deleteAddressSpace(input: $a)
+  mutation delete_as($as: [ObjectMeta_v1_Input!]!) {
+    deleteAddressSpaces(input: $a)
```
Comment (k-wall) [inline console/console-init/ui/src/graphql-module/queries/AddressSpace.ts:13]: there's a typo in the variable name.
```suggestion
    deleteAddressSpaces(input: $as)
```

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=shared.standard.web.FirefoxConsoleTest#testFilterAddressSpace crc

**Pull request merged**